Sources/FuzzilliCli/main.swift

```diff
@@ -217,6 +220,14 @@ if exportStatistics && storagePath == nil {
     configError("--exportStatistics requires --storagePath")
 }
 
+if args.double(for: "--statisticsExportInterval") == nil || statisticsExportInterval <= 0.0 {
+    configError("statisticsExportInterval needs to be > 0.0")
+}
+
+if args.double(for: "--statisticsExportInterval") != nil && !exportStatistics  {
```

Comment:
Nit: maybe just `args.has("--statisticsExportInterval")` for better readability?